### PR TITLE
Feature custom error response

### PR DESCRIPTION
**Pull request by fillup**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +45,28 @@ module "hugosite" {
...
+```hcl
+module "hugosite" {
+  source              = "github.com/fillup/terraform-hugo-s3-cloudfront"
+  aliases             = ["www.domain.com", "domain.com"]
+  bucket_name         = "www.domain.com"
+  cert_domain         = "*.domain.com"
+  deployment_user_arn = "arn:aws:iam::111122223333:person"
+  default_root_object = null
+  error_document      = "index.html"
+  custom_error_response = set(object({
```
Comment (briskt) [inline README.md]: Does this need to be changed as we found a few minutes ago?

File: README.md [hunk 2]
```diff
@@ -42,6 +45,28 @@ module "hugosite" {
...
+a 404 error to the browser. Here is an example of replacing 404 errors with 200 OK responses. 
+
+```hcl
+module "hugosite" {
+  source              = "github.com/fillup/terraform-hugo-s3-cloudfront"
+  aliases             = ["www.domain.com", "domain.com"]
+  bucket_name         = "www.domain.com"
+  cert_domain         = "*.domain.com"
+  deployment_user_arn = "arn:aws:iam::111122223333:person"
+  default_root_object = null
```
Comment (briskt) [inline README.md:60]: Not sure the correct syntax here, but the upgrade didn't like `null`

**Review by briskt — APPROVED** [review]

**Review by Baggerone — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (fillup) [reply README.md:60]: it's working fine in my tests this way, so I'll leave it

File: README.md (hunk 1 above)
Comment (fillup) [reply README.md]: yep, fixed.

**Pull request merged**